### ISSUE TITLE
Catch key events in VrEventListener (FULLSCREENMODE - Headset)

### ISSUE THREAD

**Issue by koumakis** [googlevr/gvr-android-sdk#239]
I would like to monitor different type of key events while the app is in FullScreenMode.  Using the SimpleVrPanoramaActivity and the onClick() in ActivityEventListener works well with Cardboard V2 (conductive touch screen button) or when you tap the screen. 
But what about other headets (e.g. my FreeFly VR) which use Glide/Bluetooth controllers? 
KeyEvent.Callback events are functional in SimpleVrPanoramaActivity only if the app is not in fullscreenmode. In fullscreenmode the log reports "_InputEventReceiver: Attempted to finish an input event but the input event receiver has already been disposed._" and the event is not catched by the ActivityEventListener even if I implement KeyEvent.Callback at the ActivityEventListener class. 
In short, how can I implement the KeyEvent.Callback in VrEventListener ?

**Comment by nathanmartz** [comment]
Hi. Bluetooth controllers are not a part of the official cardboard spec, so you we can't provide support for them.
